src/chart/Sankey.tsx

```diff
@@ -705,6 +715,7 @@ export class Sankey extends PureComponent<Props, State> {
         className={classNames('recharts-wrapper', className)}
         style={{ ...style, position: 'relative', cursor: 'default', width, height }}
         role="region"
+        aria-label={this.wrapperAriaLabel}
       >
         <Surface {...attrs} width={width} height={height}>
```

Comment:
So, we can pass props directly to the Surface, such as ariaLabel but we can now also pass an ariaLabel to the wrapping div? 

How would those interact? Would they be redundant to another? Should users prefer one way over the other?

Reply:
+1